packages/sdk/src/swap/sdk.ts

```diff
@@ -425,7 +430,19 @@ export class SwapSDK {
       'Affiliate brokers are supported only when setting a broker account',
     );
 
-    return this.trpc.encodeVaultSwapData.mutate(vaultSwapRequest);
+    const res = await this.apiClient.encodeVaultSwapData({
+      body: { ...vaultSwapRequest, network: this.options.network },
```

Comment:
do we need to replace bigints here?

Reply:
no, everything here is safe to serialize, i tested it against a local version of the swap server